lib/facter.rb.in

```diff
@@ -39,7 +39,7 @@ module Facter
       end
       require "#{facter_dir}/${LIBFACTER_INSTALL_DESTINATION}/libfacter.so"
     rescue LoadError
-      raise LoadError.new('libfacter was not found. Please make sure it was installed to the expected location.')
+      raise LoadError, "libfacter was not found. Please make sure it was installed to the expected location.\n" + ($!.message || ''), $!.backtrace
```

Comment:
Looks like this message isn't localized... should it be @Magisus ?

Reply:
I don't know if we have tooling to localize this ruby shim :/

Really this logic probably needs to end up in Puppet somehow, with less dependency on a hardcoded install path for libfacter.so

Reply:
We definitely don't have the tooling to translate this, unfortunately.

Reply:
Cool - thanks for the confirmation. Had a feeling multiple embedded languages in a single project were going to be tricky for the tooling.